packages/dd-trace/src/appsec/telemetry/api/manager.js

```diff
@@ -0,0 +1,124 @@
+'use strict'
+
+const dc = require('../../../../../diagnostics_channel')
+const log = require('../../../log')
+const { sendData } = require('../../../telemetry/send-data')
+
+const telemetryStartChannel = dc.channel('datadog:telemetry:start')
+const telemetryStopChannel = dc.channel('datadog:telemetry:stop')
+
+module.exports = class TelemetryManager {
+  constructor (reqType) {
+    this.reqType = reqType
+    this.initialized = false
+    this.providers = new Set()
+
+    this._onTelemetryStart = (msg) => {
```

Comment:
Does it worth to add a comment explaining that these method are defined here because they are used in subscribe/unsubscribe methods?

Reply:
added